tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/Debugger/ProbesTests.cs

```diff
@@ -45,20 +41,39 @@ public ProbesTests(ITestOutputHelper output)
         SetServiceVersion("1.0.0");
     }
 
-    public static IEnumerable<object[]> ProbeTests()
+    public static IEnumerable<object[]> AllProbeTestTypes()
     {
-        return DebuggerTestHelper.AllProbeTestTypes();
+        var testTypes = typeof(IRun)
+                       .Assembly.GetTypes()
+                       .Where(
+                            t => t.GetInterface(nameof(IRun)) != null ||
+                                 t.GetInterface(nameof(IAsyncRun)) != null);
+
+        foreach (var testType in testTypes)
+        {
+            yield return new object[] { new ProbeTestDescription() { IsOptimized = false, TestType = testType } };
+            // todo uncomment when 'Optimized' tests are supported
```

Comment:
TODO comment reminder

Reply:
Is it supported already?
I'll be happy to remove the comment.